activities.el

```diff
@@ -66,6 +66,7 @@
 (require 'map)
 (require 'persist)
 (require 'subr-x)
+(require 'vc-annotate)
```

Comment:
OT1H this seems like an elegant reuse of code; OTOH many users would never load this library otherwise (myself being one of them).  Is there another way we could do the equivalent, while still reusing some face/color definitions?

Reply:
I've used them before in other packages.  In fact I know about these colors because a better color-age dynamic range mapping was the first contribution I made to emacs (in a prior millennium).  We could rip the colors out of there and duplicate, but that colormap has been around long enough that I wonder if anyone has customized it.  It is used for blame functionality, so quite similar in spirit.  

It's trivial to select a precise color based on fractional age, using `(color-hsl-to-rgb)`, perhaps also giving users the ability to pick a lightness/saturation value in a defcustom.   Thoughts?

E.g. 
<img width="386" alt="image" src="https://github.com/alphapapa/activities.el/assets/93749/abcbcb8f-e410-450e-9e33-4b0610a7dac2">

Reply:
I'd suggest perhaps using this function I wrote for `ement`, which does something very similar but a bit more nuanced: https://github.com/alphapapa/ement.el/blob/9f6cbb8b406160c950d8932d66f3629aba091ec8/ement-room-list.el#L849-L890

Reply:
I eliminated vc-annotate, and went with a simple blend from a specified old to new color, and a blend fraction with the default foreground.